cpp-package/example/resnet.cpp

```diff
@@ -43,7 +46,7 @@ Symbol getConv(const std::string & name, Symbol data,
                                   kernel, num_filter, stride, Shape(1, 1),
                                   pad, 1, 512);
 
-  Symbol bn = BatchNorm(name + "_bn", conv, 2e-5, bn_momentum, false);
+  Symbol bn = BatchNorm(name + "_bn", conv, BN_GAMMA, BN_BETA, 2e-5, bn_momentum, false);
```

Comment:
Gamma and beta are learnable parameters that can be initialized automatically by initializer, but op.h genereator can't distinguish parameters from hyperparameters currently, as mxnet doesn't provide such information. In older versions, gamma and beta were not declared explicitly. We can use `Operator("BatchNorm")` here to avoid specifying gamma and beta before figuring out how to detect parameters.